js/freewheel.js

```diff
@@ -304,7 +305,7 @@ OO.Ads.manager(function(_, $) {
     var _setAdRequestTimeout = _.bind(function(callback, duration){
       if (adRequestTimeout) {
         var error = "Ad Request Timeout already exists - bad state";
-        fw_onError(error);
+        fw_onError(null, error);
```

Comment:
if it is possible to change the original without breaking it, yes, otherwise no. Because if we don't use that param and it isn't required (aka sdk required)  then we should remove the param so it is less confusing.